compaction/compaction.hh

```diff
@@ -85,12 +85,29 @@ struct compaction_data {
     }
 };
 
-struct compaction_result {
-    std::vector<sstables::shared_sstable> new_sstables;
+struct compaction_counters {
```

Comment:
We usually call these kind of structs `stats`, I think `compaction_stats` would be a better name.